### ISSUE TITLE
Documantation for using STIR/SHAKEN features needed

### ISSUE THREAD

**Issue by kernelpanic700** [signalwire/freeswitch#1237]
Documentation for using STIR/SHAKEN features in mod_sofia configuration and dialplan is needed.  
By now almost no information about STIR/SHAKEN implementation in FS 1.10+

**Describe the solution you'd like**
Write at least minimal doc

**Package version or git hash**
- Version of FreeSWITCH 1.10.7

**Comment by ynasida** [comment]
Yes, there is not docs really.  I am trying to  create a signed Identity header using Freeswitch. I have all stir-shaken param in sofia.conf according instructions we have. I also have set channel variable sip_stir_shaken_attest=A , but... nothing happens. Freeswitch just adds this new variable and does nothing.  If I try to use app sofia_stir_shaken_vs - it tries to verify Identity but I do not see any Freeswitch activity when I try to sign the call.

**Comment by md-riaz** [comment]
@ynasida I am using fusionpbx, so to make this work, do i need to fetch freeswitch from signalwire git repo and build freeswitch source again ?

**Comment by ynasida** [comment]
@md-riaz it was so 2 years ago. at the moment- i dont know.
